File: app/src/main/java/com/projects/finn/domain/models/Subscription.java
package com.projects.finn.domain.models;

import androidx.annotation.Keep;

import com.google.gson.annotations.SerializedName;

@Keep
public class Subscription {
    private int id;
    @SerializedName("user_id") private String userId;
    @SerializedName("community_id") private int communityId;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public int getCommunityId() {
        return communityId;
    }

    public void setCommunityId(int communityId) {
        this.communityId = communityId;
    }
}

File: app/src/main/java/com/edufelip/finn/domain/models/Subscription.kt
package com.edufelip.finn.domain.models

import androidx.annotation.Keep
import com.google.gson.annotations.SerializedName

@Keep
data class Subscription(
    var id: Int = 0,
    @SerializedName("user_id") var userId: String? = null,
    @SerializedName("community_id") var communityId: Int = 0,
)
